avaframe/out3Plot/plotUtilsCfg.ini

```diff
@@ -41,3 +41,9 @@ contCmap = False
 unitppr = kPa
 unitpfd = m
 unitpv = $ms^{-1}$
+unitrelTh = m
```

Comment:
Should we adjust all these variables names to camelCase?
ie unitRelTh, namePpr or namePPR? not sure....

Reply:
No this won't work as they are build inside the scripts using the resultType variable that we use throughout e.g. in the simulation results naming and these area 'ppr', 'pfd' and 'pv'